src/GetFunctionsMetadataCommand.cs

```diff
@@ -31,25 +32,39 @@ protected override void ProcessRecord()
         {
             try
             {
-                if (!string.IsNullOrEmpty(FunctionsAppDirectory) && Directory.Exists(FunctionsAppDirectory))
+                // Attempt to normalize the user's input to a local path from where the cmdlet is running
+                SessionState ss = new SessionState();
+                string current = Directory.GetCurrentDirectory();
+                Directory.SetCurrentDirectory(ss.Path.CurrentFileSystemLocation.Path);
+                FunctionsAppDirectoryString = Path.GetFullPath(FunctionsAppDirectoryString);
+                Directory.SetCurrentDirectory(current);
+
+                Console.WriteLine(FunctionsAppDirectoryString);
+                if (!Directory.Exists(FunctionsAppDirectoryString))
+                {
+                    ThrowTerminatingError(new ErrorRecord(new Exception(string.Format(AzPowerShellSdkStrings.InvalidFunctionsAppDirectory, FunctionsAppDirectoryString)), "InvalidFunctionAppDirectory", ErrorCategory.ParserError, null));
+                }
+
+                DirectoryInfo FunctionsAppDirectory = new DirectoryInfo(FunctionsAppDirectoryString);
+
+                if(!IsValidFunctionsAppDirectory(FunctionsAppDirectory))
+                {
+                    ThrowTerminatingError(new ErrorRecord(new Exception(string.Format(AzPowerShellSdkStrings.MissingHostJson, FunctionsAppDirectory.FullName)), "InvalidFunctionAppDirectory", ErrorCategory.ParserError, null));
+                }
+
+                List<FunctionInformation> bindingInformations = WorkerIndexingHelper.IndexFunctions(FunctionsAppDirectory, out List<ErrorRecord> errors);
+                foreach (ErrorRecord error in errors)
+                {
+                    WriteError(error);
+                }
+
+                if (PrettyPrint.IsPresent)
```

Comment:
I forgot to mentioned yesterday that we should check with the team on the name of the parameters for this cmdlet. Do you mind pinging the Teams thread with a list of all customer facing parameters and ask them if they have any concerns about the parameter names? Thank you.

Reply:
I left a comment, but if it's OK, can we let this PR go through before we get a resolution on this? The names of these parameters are not modified in this PR and I've got other changes that are blocked on this

Reply:
Renamed parameter per conversation with Anatoli and Madhura